website/content/docs/install/index.mdx

```diff
@@ -38,32 +38,53 @@ command-line, make sure to place it somewhere on your `PATH`.
 ## Compiling from Source
 
 To compile from source, you will need [Go](https://golang.org) installed and
-configured properly (including a `GOPATH` environment variable set), as well as
-a copy of [`git`](https://www.git-scm.com/) in your `PATH`.
+configured properly, as well as a copy of [`git`](https://www.git-scm.com/) in your `PATH`.
```

Comment:
@dnephin : what does "configured properly" mean (beyond installing it)?

Reply:
Good question, that text was there before I edited.

I suspect it means "$PATH should include the path to the git binary", or "the git binary must be in $PATH".

Reply:
I think it's clearer to remove "and configured properly" since it ambiguously implies a "proper" configuration (we're fine with any kind of defaults from go installation).